### PR TITLE
WSO2-AS 6.0.0 Maven JavaDoc and source plugin improvements.

### PR DESCRIPTION
**Pull request by chirangaalwis**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -114,6 +88,11 @@
         </pluginManagement>
         <plugins>
             <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>findbugs-maven-plugin</artifactId>
```
Comment (callkalpa) [inline pom.xml]: find-bugs is already there in the root pom

File: modules/http-statistics-monitoring/pom.xml [hunk 2]
```diff
@@ -29,17 +29,6 @@
         <plugins>
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-jar-plugin</artifactId>
```
Comment (callkalpa) [inline modules/http-statistics-monitoring/pom.xml]: Why remove maven-jar-plugin?

File: pom.xml [hunk 3]
```diff
@@ -114,6 +88,11 @@
         </pluginManagement>
         <plugins>
             <plugin>
+                <groupId>org.codehaus.mojo</groupId>
```
Comment (callkalpa) [inline pom.xml]: find-bugs is already defined in the root pom?

File: modules/http-statistics-monitoring/pom.xml (hunk 2 above)
Comment (chirangaalwis) [inline modules/http-statistics-monitoring/pom.xml]: Sorry, this was a mistake I made when I intended to remove the maven-source-plugin. Fixed in next commit and maven-source-plugin is removed.

**Pull request merged**